src/full/Agda/Syntax/Parser.hs

```diff
@@ -148,8 +147,8 @@ parsePosString p pos = wrapM . return . M.parsePosString pos (parseFlags p) [nor
 
 -- | Extensions supported by `parseFile`.
 
-parseFileExts :: [String]
-parseFileExts = ".agda" : literateExts
+acceptableFileExts :: [String]
+acceptableFileExts = ".agda" : (fst <$> literateProcessors)
```

Comment:
`parseFileExts` sounds very misleading to me. It's not parsing anything, how can the function name starts with `parse`?
Its value is the file extensions which can be processed by Agda, so I renamed it into `acceptableFileExts`.